includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -4,16 +4,21 @@ class DesignSystemApiController extends WikiaApiController {
 	public function getFooter() {
 		$wikiId = $this->request->getInt( 'wikiId' );
 		$lang = $this->request->getVal( 'lang' );
-
-		if ( WikiFactory::IDtoDB( $wikiId ) === false ) {
-			throw new NotFoundApiException( "Unable to find wiki with ID {$wikiId}" );
-		}
+		$fandom = $this->request->getVal( 'fandom' );
```

Comment:
Is this approach consistent with API url?
https://github.com/Wikia/chef-repo/blob/master/cookbooks/apache/templates/default/rewritesShort.conf.erb#L166